packages/playground-server/src/node.ts

```diff
@@ -60,9 +67,8 @@ export function getNodeAddress(): string {
 export async function createMultisigFor(
   userAddress: string
 ): Promise<NodeTypes.CreateMultisigResult> {
-  if (!node) {
-    node = await createNodeSingleton();
-  }
+  if (!node) node = await createNodeSingleton();
```

Comment:
@snario This change seems unnecessary.

Reply:
It's also generally not good practice to not use braces for these conditional blocks.